config.go

```diff
@@ -15,6 +15,8 @@ type Configuration struct {
 	DatabaseName       string
 	QueueName          string
 	MongoDBURI         string
+	SignalProcessingURI string
```

Comment:
as of our last discussion, you were going to use a golang implementation that didn't require accessing an external service. 

Additionally, and this is a confusing quirk, but there are two levels of configuration... this one which controls lower level service behavior and is configured per-process, and a configuration in the database (model.Configuration) that has all of the shared configuration. These values should go there.

Reply:
I think we closed out the thread with "we'll go with whatever is best" 😅 . Can we go with a python service? There is active development on the python implementation and it is not out of the question that it may even be completely replaced by some other python algo -- ideally this would be irrelevant to cedar.

Also w.r.t. the configuration --> should these not be passed as some kind of secret to the environment for cedar's processes? The token is sensitive and probably shouldn't live in a database.

Reply:
We put all kinds of tokens in the database and there's no better way to do that at the moment. 

Also my memory was "we'll support multiple implementations, but starting with the golang one," and looking back at the design document this seems to be the state of things.